dashboard/src/actions/apps.ts

```diff
@@ -55,8 +54,11 @@ const allActions = [
 
 export type AppsAction = ActionType<typeof allActions[number]>;
 
-export function getApp(releaseName: string, namespace: string) {
-  return async (dispatch: Dispatch): Promise<void> => {
+export function getApp(
+  releaseName: string,
+  namespace: string,
+): ThunkAction<Promise<void>, IStoreState, null, AppsAction> {
```

Comment:
why don't we alias this type so it's easier to reuse? e.g. we could create a `KubeappsThunkAction<void, AppsAction>` type, the first parameter being the return of the promise and the last being the action type.

Reply:
I think it's a bit easier to understand if we leave this as `ThunkAction`. 

For example, reviewing this PR I needed to investigate what the parameters of `ThunkAction` meant. If we add `KubeappsThunkAction` first I would need to check what the parameters mean there and then how it's forwarded to `ThunkAction` and finally check the implementation of that. I would avoid that extra step if it's just to avoid typing the two parameters `IStoreState, null`.

Reply:
I do not have strong opinions, but I agree with @andresmgot that this thing is convoluted and complex enough that adding another layer of indirection will not help.

Reply:
yeah I agree that it's not that important, thanks!